app-spec/test/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
```

Comment:
ALL the devDependencies except faucet can/should come out

Reply:
done